### PR TITLE
GitHub issue and pull request templates

### PR DESCRIPTION
**Pull request by peterjc**
Comments please - we want these to be clear but also helpful and welcoming.

### CODE REVIEW

**Comment by souravsingh** [comment]
It looks good to me

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+This pull request addresses issue #...
+
+I hereby agree to dual licence this any any other previous contributions under
```
Comment (cbrueffer) [inline .github/PULL_REQUEST_TEMPLATE.md]: double "any"

**Comment by cbrueffer** [comment]
Looks good apart from the one comment.

**Comment by peterjc** [comment]
Thanks @cbrueffer - reworded

**Comment by peterjc** [comment]
CodeCode was unhappy due to a tiny drop in coverage - we must have some stochastic tests causing that.

**Pull request merged**